data/sqlutil/sql.go

```diff
@@ -182,7 +182,7 @@ func Replace(frame *data.Frame, fieldIdx int, replacer *StringFieldReplacer) err
 	if field.Type() != data.FieldTypeNullableString {
 		return fmt.Errorf("can only replace []*string vectors, vector is of type %s", field.Type())
 	}
-	//newVector := newVector(replacer.VectorType, field.vector.Len())
+	// newVector := newVector(replacer.VectorType, field.vector.Len())
```

Comment:
looks like this comment should just be removed

Reply:
@scottlepp Good point, thanks :)